reference/troubleshooting/troubleshoot-multi-stage-query-engine.md

```diff
@@ -131,7 +131,7 @@ CAST(timestampCol AS BITINT) >= longCol
 
 Default names for projections with function calls are different between v1 and v2.&#x20;
```

Comment:
Not introduced in this PR, but should we call it single-stage and multi-stage engine?

Reply:
Yes. There are tons of mentions to v1 and v2. I've pushed a commit changing most of them. If you prefer I can cherry pick that commit into another PR